src/lib/locals.js

```diff
@@ -7,9 +7,9 @@ const {
 const { generateNonce } = require("./strings");
 
 module.exports = {
-  getGTM: function (req, res, next) {
+  setLocals: function (req, res, next) {
     res.locals.gtmId = GTM_ID;
-    res.locals.scriptNonce = generateNonce();
+    res.locals.cspNonce = generateNonce();
```

Comment:
This value is used by the `govuk-frontend` template, and must be called `cspNonce`.